phylogenetic/config/dropped_strains.txt

```diff
@@ -1,69 +1,49 @@
-DENV/SPAIN/EEB17/2009
-DENV1/FRANCE/00475/2008
-DENV1/MALAYSIA/P1244/1972
-DENV1/VIETNAM/BIDV3990/2008
-DENV1/VIETNAM/BIDV992/2006
-DENV2/AUSTRALIA/QML22/2015
-DENV2/BURKINA_FASO/DAKAR2039/1980
-DENV2/BURKINA_FASO/DAKARA2022/1980
-DENV2/COTE_D_IVOIRE/DAKAR510/1980
-DENV2/COTE_D_IVOIRE/DAKAR578/1980
-DENV2/COTE_D_IVOIRE/DAKARA1247/1980
-DENV2/GUINEA/PM33974/1981
-DENV2/HAITI/DENGUEVIRUS2HOMOSAPIENS1/2016
-DENV2/MALAYSIA/DKD811/2008
-DENV2/MALAYSIA/P81407/1970
-DENV2/MALAYSIA/SAB/2015
-DENV2/NIGERIA/IBH11208/1966
-DENV2/NIGERIA/IBH11234/1966
-DENV2/NIGERIA/IBH11664/1966
-DENV2/SENEGAL/0674/1970
-DENV2/SENEGAL/DAKAR0761/1974
-DENV2/SENEGAL/DAKAR141069/1999
-DENV2/SENEGAL/DAKAR141070/1999
-DENV2/SENEGAL/DAKARD75505/1999
-DENV2/TRINIDAD_AND_TOBAGO/NA/1953
-DENV4/MALAYSIA/P215/1975
-DENV4/MALAYSIA/P514/1975
-DENV4/MALAYSIA/P731120/1973
-D2Sab2015 # miscategorized
-QML22 # miscategorized
-DAK_Ar_A1247 # sylvatic
-Dak_Ar_2039 # sylvatic
-Dak_Ar_578 # sylvatic
-DAK_Ar_510 # sylvatic
-PM33974 # sylvatic
-Dak_Ar_A2022 # sylvatic
-Dak_Ar_141069 # sylvatic
-Dak_Ar_141070 # sylvatic
-Dak_Ar_D75505 # sylvatic
-Dak_HD_10674 # sylvatic
-Dak_Ar_D20761 # sylvatic
-IBH11664 # sylvatic
-IBH11208 # sylvatic
-IBH11234 # sylvatic
-P8_1407 # sylvatic
-P75_514 # sylvatic
-P73_1120 # sylvatic
-P75_215 # sylvatic
-DKD811 # sylvatic
-ZS01/01 # metadata issue
-Vero # cell line
-MS13002673 # too divergent
-MS11011405 # too divergent
-V43257 # too divergent
-KDC0574A2_06/02/2011 # too divergent
-00178/03 # too divergent
-00759/12 # too divergent
-00988/11 # too divergent
-01113/10 # too divergent
-01224/04 # too divergent
-01231/10 # too divergent
-01488/09 # too divergent
-01542/04 # too divergent
-dev1 # too divergent
-DKE_121 # too divergent
-SENDAK_HD_10674 # sylvatic
-DENV2_1_DAK_HD_76395 # sylvatic
-DENV3/PUERTORICO/1963/PRS_228762_AC27 # too divergent
-PR_6 # too divergent
+JF260983 # DENV/SPAIN/EEB17/2009 # sylvatic according to https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3149010/
```

Comment:
This file has grown to more than a simple list of strain names, which used to be self-documenting by the filename. Without the context of this PR, it can be unclear what the contents are. 2 suggestions:

1. Add a comment at the top of the file describing the fields. Example:

    ```
    # Format: <GenBank accession> [# <strain name> <reason>]
    ```

2. Rename the file to `dropped_accessions.txt`.

Reply:
I'll add the suggested documentation as comments at the beginning of the file, thanks!

Also, you raise a valid point about the filename `dropped_strains.txt` possibly being a misnomer when we've switched to ID by accession. I ran a quick search of how this file is named in other pathogen repositories:

* seasonal-flu (ID by strain name)
	* H1N1: [outliers.txt](https://github.com/nextstrain/seasonal-flu/blob/master/config/h1n1pdm/outliers.txt)
	* H3N2: [outliers.txt](https://github.com/nextstrain/seasonal-flu/blob/master/config/h3n2/outliers.txt)
	* vic: [outliers.txt](https://github.com/nextstrain/seasonal-flu/blob/master/config/vic/outliers.txt)
* mpox: (ID by accession)
	* all: [exclude_accessions.txt](https://github.com/nextstrain/mpox/blob/master/phylogenetic/config/exclude_accessions.txt) <- I'm leaning toward this one
	* but also has a generically named [include.txt](https://github.com/nextstrain/mpox/blob/master/phylogenetic/config/hmpxv1_big/include.txt) (which gets passed to `augur filter --include`)
* measles: (ID by st. ain, but will be moving to accession)
	* [dropped_strains.txt](https://github.com/nextstrain/measles/blob/main/config/dropped_strains.txt)
* zika: (ID by accession)
	* [dropped_strains.txt](https://github.com/nextstrain/zika/blob/main/phylogenetic/config/dropped_strains.txt)
* zika-tutorial: (ID by strain name)
	* [dropped_strains.txt](https://github.com/nextstrain/zika-tutorial/blob/master/config/dropped_strains.txt)
	* [zika tutorial filter documentation](https://docs.nextstrain.org/en/latest/tutorials/creating-a-workflow.html#filter-the-sequences)
* rsv: (ID by accession)
	* [outliers.txt](https://github.com/nextstrain/rsv/blob/master/config/outliers.txt)

Of the above options and considering that the file is being passed to `augur filter --exclude`, I'm inclined toward naming it `exclude_accessions.txt`. 

Or we could lean more generic as `exclude.txt` to match the `include.txt` with the assumption that the user can pick any ID column that matches `--metadata-id-columns`. Do you have a preference?

Reply:
Thanks for surveying similar files. From the looks of it, not much thought has been put into standardizing these filenames, and it's not in the current scope of [pathogen-repo-guide](https://github.com/nextstrain/pathogen-repo-guide).

If you have the comment line at the top indicating that entries are accessions, then it's not necessary to include `accession` in the filename and might even be better to be generic so filename doesn't need to be changed every time the ID column is changed.

Either `exclude.txt` or `outliers.txt` sound good to me. If we want to consider standardizing these eventually, it might be easiest to go with `outliers.txt` since there's already some files with that name.

Reply:
If we want to standardize across pathogens, my vote would be for the more generic `exclude.txt`. Sequences can be excluded from builds for a variety of reasons other than being outliers.

Reply:
@joverlee521 I didn't realize this file is mentioned in pathogen-repo-guide. Thanks for adding https://github.com/nextstrain/pathogen-repo-guide/commit/3120df9eae7fb124a09c9f0f028738fc93185514!